From be7b5aeaf439ee9025c1b70db3cf466c14c299da Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mervin.fansler@bric.ku.dk>
Date: Thu, 17 Oct 2024 08:59:49 +0200
Subject: [PATCH] skip test 1623 if no '~' (fix #6576)

---
 inst/tests/tests.Rraw | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/inst/tests/tests.Rraw b/inst/tests/tests.Rraw
index c530b2fd1..01094fb19 100644
--- a/inst/tests/tests.Rraw
+++ b/inst/tests/tests.Rraw
@@ -8913,7 +8913,7 @@ test(1622.1, ans1, ans2)
 test(1622.2, ans1, fread(testDir("issue_1573_fill.txt"), fill=TRUE, sep=" ", na.strings=""))
 
 # fix for #989
-test(1623, fread("~"), error="File '~' is a directory")
+if (dir.exists("~")) test(1623, fread("~"), error="File '~' is a directory")
 
 # testing print.rownames option, #1097 (part of #1523)
 options(datatable.print.rownames = FALSE)
